subjects/devops/string_tokenizer/README.md

```diff
@@ -0,0 +1,91 @@
+## String tokenizer
+
+### Instructions
+
+Tokenization is the process of breaking down a string into smaller pieces, called tokens. In natural language processing, tokenization typically refers to the process of breaking down a sentence into words or breaking down a paragraph into sentences.
+
+Create a file `string_processing.py` which will have a function `tokenize(sentence)` that given a sentence will do the following:
+
+- removes all punctuation marks and special characters
```

Comment:
`remove` since before the colon it's future.

Reply:
done